crypto/evp/p_seal.c

```diff
@@ -63,9 +64,9 @@ int EVP_SealInit(EVP_CIPHER_CTX *ctx, const EVP_CIPHER *type,
         }
 
         if (EVP_PKEY_encrypt_init(pctx) <= 0
-            || EVP_PKEY_encrypt(pctx, ek[i], &keylen, key, keylen) <= 0)
+            || EVP_PKEY_encrypt(pctx, ek[i], &outlen, key, len) <= 0)
```

Comment:
I know static analysers are going to complain about the cast following this, so I'd suggest:
```suggestion
            || EVP_PKEY_encrypt(pctx, ek[i], &outlen, key, len) <= 0
            || outlen > INT_MAX)
```

Reply:
emmmmm, fixed it. compromising with the complain.